modules/virt-uploading-image-web.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-uploading-local-disk-images-web.adoc
+
+[id="virt-uploading-image-web_{context}"]
+= Uploading an image file using the web console
+
+Use the web console to upload an image file to a new Persistent Volume Claim (PVC).
+You can later use this PVC to attach the image to new virtual machines.
+
+.Prerequisites
+
+* You must have one of the following:
+** A raw virtual machine image file in either ISO or IMG format.
+** A virtual machine image file in QCOW2 format.
+* For best results, compress your image file according to the following guidelines before you upload it:
+** Compress a raw image file by using `xz` or `gzip`.
++
+[NOTE]
+====
+Using a compressed raw image file results in the most efficient upload.
+====
+** Compress a QCOW2 image file by using the method that is recommended for your client:
+*** If you use a Linux client, _sparsify_ the QCOW2 file by using the
+link:https://libguestfs.org/virt-sparsify.1.html[virt-sparsify] tool.
+*** If you use a Windows client, compress the QCOW2 file by using `xz` or `gzip`.
+
+.Procedure
+
+. From the side menu, click *Storage* -> *Persistent Volume Claims*.
+
+. Click the *Create Persistent Volume Claim* drop-down list to expand it.
+
+. Click *With Data Upload Form* to open the *Upload Data to Persistent Volume Claim* page.
+
+. Click *Browse...* to open the file manager and select the image that you want to upload, or drag the file into the *Drag a file here or browse to upload* field.
+
+. Optional: Set this image as the default image for a specific operating system.
+
+.. Select the *Attach this data to a Virtual Machine operating system* check box.
+
+.. Select an operating system from the list.
+
+. Enter a name for the PVC in the *Persistent Volume Claim Name* field. This name must be unique within the project.
```

Comment:
The PVC name is filled automatically and not allowed to change based on current implementation.

Reply:
Thanks for letting me know! I have changed this step to `. The *Persistent Volume Claim Name* field is automatically filled and cannot be changed. Take note of the assigned name for the PVC.`

Any other concerns? Thanks, Guohua.